examples/reshare/src/application/actor.rs

```diff
@@ -26,7 +26,9 @@ where
     V: Variant,
 {
     context: ContextCell<E>,
-    mailbox: mpsc::Receiver<Message<H, C, V>>,
+    mailbox: mpsc::Receiver<Message<H>>,
+
+    _phantom: PhantomData<(C, V)>,
```

Comment:
Not necessary, but could move the `C` + `V` bounds to `start` / `run`, since they aren't used on the type itself anymore.